src/constants.tsx

```diff
@@ -84,6 +84,8 @@ export const REPORT = 'report';
 export const BACKEND_CALLBACK_URL = '/fe/oauth/callback/opensrp';
 export const BACKEND_CALLBACK_PATH = '/fe/oauth/callback/:id';
 export const REACT_CALLBACK_PATH = '/oauth/callback/:id';
+export const STUDENTS_LIST_URL = '/students';
+export const UPLOAD_STUDENT_CSV_URL = '/students/upload';
```

Comment:
Should we lock ourselves in to call this `students`?

Reply:
Let's not have student here, lets have client instead.  My thinking is that since this is not exposed to the user, we don't need to make it Eswatini-specific.